quickstart/201-synapse-secure/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "name" {
+  type        = string
+  description = "Name of the deployment"
+}
+
+variable "environment" {
+  type        = string
+  description = "Name of the environment"
+  default     = "dev"
+}
+
+variable "location" {
+  type        = string
+  description = "Location of the resources"
+  default     = "East US"
+}
+
+variable "aad_login" {
+  description = "AAD login"
+  type = object({
+    name      = string
+    object_id = string
+    tenant_id = string
+  })
+  default = {
```

Comment:
Apologies for missing this on the first review but this should not have a default unless, you can provide values that will allow the configuration to succeed. Please remove the default.

Reply:
@grayzu done